msticpy/config/ce_azure.py

```diff
@@ -37,6 +37,9 @@ class CEAzure(CESimpleSettings):
     <li>msi - use Managed Service Identity credentials, if available</li>
     <li>cli - use Azure CLI credentials, if available</li>
     <li>interactive - use interactive browser authentication</li>
+    <li>vscode" - to use VSCode credentials</li>
```

Comment:
I think we need to add these options to the config yaml right?
in resources\mpconfig_defaults.yaml
```yaml
Azure:
  cloud: enum(required=False, options=[global; cn; usgov; de], default=global)
  auth_methods: m_enum(required=False, options=[env; msi; cli; interactive; vscode; powershell; cache], default=["cli"; "msi", "interactive"])
```
Not sure if we want to change the defaults.